SSTTP-1150 and STTP-1154

cavster opened the pull request:
Changed Instalment  summary page and changed section to have no padding in css and updated print message

cavster committed:
Updated installment summary page and deleted unused messages

Changed  print to Print

Smudgey reviewed (changes requested):
The layout of the page doesn't quite match the prototype.

The details drop down section is underlined when it shouldn't be. There is a grey bar to the left of the text this details section opens that is also missing.

There needs a bit more space between the details section and the grey bullet point block of text at the bottom.

Whilst not introduced in this PR, There is a minor error in the text at the bottom, in the following text there is a comma after the word plan. This needs to be removed.

"If the interest rate changes during your instalment plan, we’ll settle the amount of interest you owe in your final payment."

cavster committed:
Updated html as per prototype for the application complete page

Updated installment plan summary and removed section padding set to zero in css

Added in test to banner on application complete page

Added in custom css for gap in  instalment summry

removed space between header and banner and changed message on application complete page

Smudgey commented:
There is a big space at the top of the Instalment Summary page between the header and the start of the payment schedule.

The application complete page does not match up with the prototype. There is text missing from the green box at the top, text under 'What happens next?' is different and the green button text is incorrect.

![appl_comp_1154](https://cloud.githubusercontent.com/assets/7337624/23861688/b114c216-0802-11e7-9f44-42fa087e96e9.png)

nkaraolis commented:
Green button text is still incorrect
What happens next? heading is too big compared the prototype
Text under what happens next is still incorrect

nkaraolis commented:
Green button text still incorrect and there are merge conflicts but heading text is all good now

cavster committed:
Added in us on green button on last page

nkaraolis commented:
All code looks okay now, just run a quick imports optimisation when you fix the merge conflicts

cavster committed:
cleaned up imports in application_complete and instalment_plan_summary

cavster committed:
erge with master latest

nkaraolis commented on conf/messages:
```diff
@@ -274,11 +274,21 @@ ssttp.you-need-to-file.other-tax=If you’re unsure how to proceed for any reaso
 ssttp.you-need-to-file.other-tax.number=0300 200 3504
 
 ssttp.arrangement.instalment-summary.title=Your instalment plan summary
+<<<<<<< HEAD
```
The merge has left in the conflict lines and as such has stopped the branch from compiling so these will need removing as well adding in loads of duplicated messages

nkaraolis commented on app/views/selfservicetimetopay/arrangement/application_complete.scala.html:
```diff
@@ -31,21 +29,23 @@
+          <p><a href="javascript:window.print()" class="js-visible no-print float--right  print-link js-visible trackedPrintLink openOnPrint">@Messages("ssttp.arrangement.instalment-summary.print")</a></p>
+          <h2>@Messages("ssttp.arrangement.complete.next")</h2>
     <p>@Messages("ssttp.arrangement.complete.summary")</p>
       <button onclick="window.print()" class="js-visible no-print button--secondary js-visible trackedPrintLink openOnPrint">@Messages("ssttp.arrangement.complete.print")</button>
```
Do we need the window.print()? The button just needs to be a dropdown to display the hidden data so this might not be necessary

nkaraolis reviewed (changes requested):
Read code comments relating to merge errors and print button

cavster committed:
removed unused button and fixed messages file

cavster committed:
slight spelling mistake

cavster committed:
Merge branch 'master' into colmNick/SSTTP-1150/ChangeInstallmentSummaryPage

Smudgey commented on app/views/selfservicetimetopay/arrangement/application_complete.scala.html:
```diff
@@ -31,21 +29,22 @@
+  <section>
     <details>
-        <summary>@Messages("ssttp.arrangement.complete.payment-details.summary")</summary>
+        <summary style="width: 200px" class = "button button--secondary gutter-left">@Messages("ssttp.arrangement.complete.payment-details.summary")</summary>
```
This should be extracted to the ssttp css file

Smudgey commented on conf/messages:
```diff
@@ -340,8 +340,7 @@ ssttp.direct-debit.guarantee.title=Direct Debit Guarantee
 ssttp.direct-debit.guarantee.list.3=If an error is made in the payment of your Direct Debit by HM Revenue & Customs or your bank or building society you are entitled to a full and immediate refund of the amount paid from your bank or building society.
-ssttp.direct-debit.guarantee.list.4=If you receive a refund you are not entitled to, you must pay it back when HM Revenue & Customs asks you to.
-ssttp.direct-debit.guarantee.list.5=You can cancel a Direct Debit at any time by simply contacting your bank or building society. Written confirmation may be required. Please also notify us.
+ssttp.direct-debit.guarantee.list.4=If you receive a refund you are not entitled to, ssttp.arrangement.instalment-summary.printly contacting your bank or building society. Written confirmation may be required. Please also notify us.
```
This line contains ssttp.arrangement.instalment-summary within the text message? Should the direct-debit guarantee even be changing? list.5 seems to have been deleted

cavster committed:
added in deleted message and moved hard codeing style to css

nkaraolis commented on conf/messages:
```diff
@@ -341,7 +341,7 @@ ssttp.direct-debit.guarantee.list.1=This Guarantee is offered by all banks and b
 ssttp.direct-debit.guarantee.list.2=If there are any changes to the amount, date or frequency of your Direct Debit HM Revenue & Customs will notify you 10 working days in advance of your account being debited or as otherwise agreed. If you request HM Revenue & Customs to collect a payment, confirmation of the amount and date will be given to you at the time of the request.
 ssttp.direct-debit.guarantee.list.3=If an error is made in the payment of your Direct Debit by HM Revenue & Customs or your bank or building society you are entitled to a full and immediate refund of the amount paid from your bank or building society.
 ssttp.direct-debit.guarantee.list.4=If you receive a refund you are not entitled to, ssttp.arrangement.instalment-summary.printly contacting your bank or building society. Written confirmation may be required. Please also notify us.
```
The text for this message doesn't match the prototype

nkaraolis commented on public/css/self-service-time-to-pay.css:
```diff
@@ -38,10 +38,17 @@
     float:left;
     white-space: nowrap;
 }
+.ssttp-summary{
```
Is this empty class needed?

cavster replied:
Yes

Commit:
Fixed direct debit message and removed redundant css
